packages/react-core/src/components/Menu/MenuContainer.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { Popper, PopperProps } from '../../helpers/Popper/Popper';
+
+export interface MenuContainerProps {
+  /** Menu to be rendered */
+  menu: React.ReactElement<any, string | React.JSXElementConstructor<any>>;
```

Comment:
I could see it going either way, but I think the name 'menuContainer' makes me lean toward advocating for passing the menu in via `children`.

Reply:
After tinkering with it a bit, `children` may not work very well because of the type that Popper is expecting for the popper menu. It works out for Select/Dropdown because they create the menu internally, but a passed children in MenuContainer would be a ReactNode instead of the JSXConstructor that Popper needs and errors out.